backends/advanced/.env.template

```
@@ -58,5 +58,49 @@ LANGFUSE_SECRET_KEY=
+# 1. Go to Google Account → Security → 2-Step Verification
+# 2. Scroll to "App passwords" → Generate password for "Mail"
+# 3. Use the 16-character password below (no spaces)
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USERNAME=your-email@gmail.com
+SMTP_PASSWORD=your-app-password-here
+SMTP_USE_TLS=true
+
+# Email sender information
+FROM_EMAIL=noreply@chronicle.ai
+FROM_NAME=Chronicle AI
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)